catalyst-gateway/bin/src/service/common/types/cardano/cip19_address.rs

```diff
@@ -42,6 +46,10 @@ const PATTERN: &str = concatcp!(
     PROD,
     "|",
     TEST,
+    "|",
+    STAKE_PROD,
```

Comment:
If we want to change this to be "Address" instead of just Shelley address, we need to modify the description, title etc.

Also
```
/// Length of the encoded address (for type 0 - 3).
const ENCODED_STAKED_ADDR_LEN: usize = 98;
/// Length of the encoded address (for type 6 - 7).
const ENCODED_UNSTAKED_ADDR_LEN: usize = 53;
```
Eventhough stake address can use the `ENCODED_UNSTAKED_ADDR_LEN`, it miscommunicate the meaning. It is called "unstaked" because type 6-7 dont have delegation part, but for stake address, it does have the stake part